src/store/filters-slice.test.tsx

```diff
@@ -0,0 +1,76 @@
+import reducer, {
+  initialState,
+  genreChanged,
+  sortingChanged,
+  searchChanged,
+  selectGenre,
+  selectSorting,
+  selectSearch,
+} from './filters-slice';
+
+test('should return the initial state', () => {
+  // @ts-ignore
+  expect(reducer(undefined, {})).toEqual(initialState);
+});
+
+test('should set genre', () => {
+  const previousState = initialState;
+
+  expect(reducer(previousState, genreChanged('Comedy'))).toEqual({
```

Comment:
Just suggestion. You could create a variable with the new state and pass it into `toEqual`, better to read.